certgen/csr_sign.c

```diff
@@ -50,7 +50,6 @@ static int do_csrsign(int argc, char** argv)
 {
     int ret = 0;
     int type = ECC_TYPE;
-    const char* typeStr = "ecc";
```

Comment:
FYI: Artifact from the wolfTPM CSR example: https://github.com/wolfSSL/wolfTPM/tree/master/examples/csr